recipes-bsp/u-boot/u-boot-iot2050-2019.01-ti-sdk.inc

```diff
@@ -15,27 +16,27 @@ SRC_URI += " \
-    file://0001-feat-add-iot2050-platform-support.patch \
-    file://0002-am654x-frequency-update-for-both-arm-clusters-and-bu.patch \
-    file://0003-iot2050-turn-on-red-led-when-something-goes-wrong.patch \
-    file://0004-am65x-fix-usb-hub-issue-for-disabling-charge-detect.patch \
-    file://0005-am654x-remove-dependency-of-TI_SECURE_DEV_PKG.patch \
-    file://0006-mmc-change-SD-to-mmc0-and-EMMC-Flash-to-mmc1.patch \
-    file://0007-feat-add-scripts-to-select-fdt-name.patch \
-    file://0008-feat-add-UUID-checking-before-run-distro_bootcmd.patch \
-    file://0009-setting-the-rj45-port-led-behavior.patch \
-    file://0010-fix-rproc-init-error-in-u-boot.patch \
-    file://0011-feat-catch-up-100M-full-duplex-in-u-boot.patch \
-    file://0012-feat-add-the-flash-protect-for-winbond-flash.patch \
-    file://0013-bugfix-set-gpio-direction-output-show-error.patch \
-    file://0014-feat-add-config-for-skipping-certificate-verificatio.patch \
-    file://0015-feat-enable-verified-boot.patch \
-    file://0016-feat-add-sysfw-19.12-support.patch \
-    file://0017-add-the-signature-support-for-fit-image-add-the-supp.patch \
-    file://0018-feat-set-sdhci0-clock-frequency-to-142.86MHz.patch \
+    file://2019.01-ti-sdk/0001-feat-add-iot2050-platform-support.patch \
+    file://2019.01-ti-sdk/0002-am654x-frequency-update-for-both-arm-clusters-and-bu.patch \
+    file://2019.01-ti-sdk/0003-iot2050-turn-on-red-led-when-something-goes-wrong.patch \
+    file://2019.01-ti-sdk/0004-am65x-fix-usb-hub-issue-for-disabling-charge-detect.patch \
+    file://2019.01-ti-sdk/0005-am654x-remove-dependency-of-TI_SECURE_DEV_PKG.patch \
+    file://2019.01-ti-sdk/0006-mmc-change-SD-to-mmc0-and-EMMC-Flash-to-mmc1.patch \
+    file://2019.01-ti-sdk/0007-feat-add-scripts-to-select-fdt-name.patch \
+    file://2019.01-ti-sdk/0008-feat-add-UUID-checking-before-run-distro_bootcmd.patch \
+    file://2019.01-ti-sdk/0009-setting-the-rj45-port-led-behavior.patch \
+    file://2019.01-ti-sdk/0010-fix-rproc-init-error-in-u-boot.patch \
+    file://2019.01-ti-sdk/0011-feat-catch-up-100M-full-duplex-in-u-boot.patch \
+    file://2019.01-ti-sdk/0012-feat-add-the-flash-protect-for-winbond-flash.patch \
+    file://2019.01-ti-sdk/0013-bugfix-set-gpio-direction-output-show-error.patch \
+    file://2019.01-ti-sdk/0014-feat-add-config-for-skipping-certificate-verificatio.patch \
+    file://2019.01-ti-sdk/0015-feat-enable-verified-boot.patch \
+    file://2019.01-ti-sdk/0016-feat-add-sysfw-19.12-support.patch \
+    file://2019.01-ti-sdk/0017-add-the-signature-support-for-fit-image-add-the-supp.patch \
+    file://2019.01-ti-sdk/0018-feat-set-sdhci0-clock-frequency-to-142.86MHz.patch \
+    file://2019.01-ti-sdk/iot2050-uboot-ti-sdk-rules \
     file://atf/0001-feat-add-atf-support-to-iot2050-platform.patch;patchdir=${WORKDIR}/atf \
     file://optee/0001-feat-add-optee-support-to-iot2050-platform.patch;patchdir=${WORKDIR}/optee \
-    file://iot2050-uboot-ti-sdk-rules \
```

Comment:
Why delete this files? I don't think binman is supported in these patches.

Reply:
Haven't delete these patches,just archive into 2019.01-ti-sdk folder